middleware/error-handler.js

```diff
@@ -1,19 +1,16 @@
-const { CustomAPIError } = require('../errors')
 const { StatusCodes } = require('http-status-codes')
 const errorHandlerMiddleware = (err, req, res, next) => {
   
   let customError = {
-
+    // Set default
     statusCode: err.statusCode || StatusCodes.INTERNAL_SERVER_ERROR,
-    msg:err.message|| 'Something went wrong try again later'
+    msg:err.message|| 'Something went wrong try again later',
   }
-
-  //if (err instanceof CustomAPIError) {
-  //  return res.status(err.statusCode).json({ msg: err.message })
-  //}
-
+  
   if(err.name === 'ValidationError') {
-    customError.msg = Object.values(err.errors).map((item) => item.message).json(', ')
+    customError.msg = Object.values(err.errors)
+    .map((item) => item.message)
+    .json(',')
```

Comment:
You should use the join function here and not json.

Reply:
I have fixed it